Source/WebCore/platform/audio/AudioSession.cpp

```diff
@@ -135,6 +137,8 @@ void AudioSession::beginInterruption()
 
 void AudioSession::endInterruption(MayResume mayResume)
 {
+    RELEASE_LOG_ERROR(WebRTC, "AudioSession::endInterruption");
```

Comment:
Ditto

Reply:
Ditto

Reply:
This should be removed, I did it in the new patch